core-services/prow/02_config/_config.yaml

```diff
@@ -1121,6 +1112,13 @@ tide:
     - openshift/tektoncd-pipeline-operator
     - openshift/tektoncd-catalog
     - openshift/tektoncd-triggers
+    - openshift/knative-build
```

Comment:
do we really need knative-build ? :-)  /cc @vdemeester

Reply:
nop knative-build is no more, I feel this repository should be archived, shouldn't it ?